deno.lock

```diff
@@ -87,9 +87,11 @@
       "npm:mongodb@6.1.0": "npm:mongodb@6.1.0",
       "npm:postcss-import@16.1.0": "npm:postcss-import@16.1.0_postcss@8.4.38",
       "npm:postcss@8.4.38": "npm:postcss@8.4.38",
+      "npm:postcss@8.4.39": "npm:postcss@8.4.39",
       "npm:preact": "npm:preact@10.22.0",
       "npm:preact-render-to-string@6.5.5": "npm:preact-render-to-string@6.5.5_preact@10.22.0",
       "npm:preact@10.22.0": "npm:preact@10.22.0",
+      "npm:preact@10.22.1": "npm:preact@10.22.1",
```

Comment:
Let's stay on `10.22.1`, need to release a new Fresh 1.x version to make Fresh work with latest Preact